src/java.base/share/classes/java/lang/ScopedValue.java

```diff
@@ -157,14 +157,19 @@
  * {@snippet lang=java :
  *     private static final ScopedValue<String> NAME = ScopedValue.newInstance();
 
- *     ScopedValue.runWhere(NAME, "duke", () -> {
- *         try (var scope = new StructuredTaskScope<String>()) {
+ *     ScopedValue.where(NAME, "duke").run(() -> {
+ *         // @link substring="open" target="StructuredTaskScope#open()" :
+ *         try (var scope = StructuredTaskScope.open()) {
  *
```

Comment:
You'll need to revert this line as that is the updated STS API so will be update with its next JEP.

Reply:
Make sure to run jdk:tier1 and jdk:tier2 as that will run the tests that use this API.